pleskdistup/common/src/util.py

```diff
@@ -9,31 +9,63 @@
 # Returns standard output
 def logged_check_call(cmd: typing.Union[typing.Sequence[str], str], **kwargs) -> str:
     log.info(f"Running: {cmd!r}. Output:")
+    stdout = []
+
+    def proc_stdout(line: str) -> None:
+        log.info(line, to_stream=False)
+        stdout.append(line)
+
+    def proc_stderr(line: str) -> None:
+        log.err(line)
+
+    exit_code = exec_get_output_streamed(cmd, proc_stdout, proc_stderr, **kwargs)
+    if exit_code != 0:
+        log.err(f"Command {cmd!r} failed with return code {exit_code}")
+        raise subprocess.CalledProcessError(returncode=exit_code, cmd=cmd, output="\n".join(stdout))
+
+    log.info(f"Command {cmd!r} finished successfully")
+    return "\n".join(stdout)
```

Comment:
Nitpick: note that in general that would not be equivalent to the real process stdout (due to some potentially missing `\n` and/or whitespace characters).

Reply:
It seems minor as it was from the previous impl and noone complained about it (yet?)

Reply:
Not removing last newline - it's `"".join(stdout)` now